src/utils/utils.js

```diff
@@ -16,7 +18,27 @@ function checkHTTPS(ALLOW_HTTP) {
   }
 }
 
+async function waitForFunds(web3, address, cb) {
+  const balance = web3.utils.toBN(await web3.eth.getBalance(address))
+
+  promiseRetry(
+    async retry => {
+      const newBalance = web3.utils.toBN(await web3.eth.getBalance(address))
+      if (newBalance.eq(balance)) {
```

Comment:
you are looking for any difference in balances. Does it make sense to resume only if balance is enough to send particular transaction (gas usage * gas price)?

Reply:
Fixed, thanks.